aalto/paniikki.rst

```diff
@@ -72,19 +72,19 @@ Hardware
 Software
 ========
 First thing first, you DO NOT have a sudo right in Aalto classroom
-machines and you never will. We provide mostly used SW and if you need
-more you could inquire via servicedesk@aalto.fi.  We try to have good
-base software.
+machines and you never will. We provide the most used SW and if you need
+more you could inquire via servicedesk@aalto.fi.  We try to have a good
+base software that covers most people's needs.
 
 .. csv-table::
    :header-rows: 1
    :delim: |
 
    What?  |  How?
 
-   Python via Anaconda | ``module load anaconda3`` (anaconda2 for Python2)
+   Python via Anaconda | ``module load anaconda3`` (anaconda3 for Python2)
```

Comment:
What exactly is wrong?  I see confusion between "2" and "3" but do you think it should be changed to 2 only or 3 only or both or ... ?

We should be more welcoming to pull requests.  We can also just fix it right in the web interface with suggestions.

Reply:
```suggestion
   Python via Anaconda | ``module load anaconda3`` (anaconda3 for Python3)
```

Reply:
> What exactly is wrong? I see confusion between "2" and "3" but do you think it should be changed to 2 only or 3 only or both or is anaconda3 not the name or ... ?

Well, with the patch it says that one should use the anaconda3 module to get a python2 environment, which is incorrect.

Then again, since python2 is EOL in a few weeks, should we just remove any documentation and pointers about python2 and just have docs for python3?

> We should be more welcoming to pull requests. We can also just fix it right in the web interface with suggestions.

Sorry, I didn't mean this to dissuade contributions, just help the submitter to improve the submission so it can be accepted.

Reply:
Even the old version mixed python2 and python3, in the same line, making me wonder if it should go more forward or backwards.  Anyway, committing and merging.